internal/cloud/realm/secrets.go

```diff
@@ -27,10 +28,11 @@ func (c *client) Secrets(groupID, appID string) ([]Secret, error) {
 	if resErr != nil {
 		return nil, resErr
 	}
+
 	if res.StatusCode != http.StatusOK {
 		return nil, api.ErrUnexpectedStatusCode{"secrets", res.StatusCode}
 	}
-	defer res.Body.Close()
```

Comment:
we definitely still want this here, as we read from the `res.Body` next here

Reply:
oops this was a mistake